Source/Core/Common/Config/Config.cpp

```diff
@@ -17,6 +18,7 @@ using Layers = std::map<LayerType, std::shared_ptr<Layer>>;
 static Layers s_layers;
 static std::list<ConfigChangedCallback> s_callbacks;
 static u32 s_callback_guards = 0;
+static std::atomic<u32> s_config_version = 0;
```

Comment:
I'm not sure this will ever be an issue... but maybe use u64 just so we never have to worry about overflowing?